etc/scripts/osikeys.py

```diff
@@ -0,0 +1,44 @@
+import urllib.request
+import licensedcode.index
+import json
+from licensedcode.cache import get_index
+from licensedcode.models import load_licenses
+from licensedcode.models import Rule
+from licensedcode.models import rules_data_dir
+
+def findOSIMap(license):
```

Comment:
nitpicking... we prefer to use lowercase snake case rather than camel case for identifiers (e.g. `find_osi_map`) here and elsewhere